extensions/tasks/NLUTestV0/index.ts

```diff
@@ -2,12 +2,14 @@ import * as tl from 'azure-pipelines-task-lib/task';
 import * as tr from 'azure-pipelines-task-lib/toolrunner';
 import * as path from 'path';
 import * as utilities from 'nlu-devops-common/utilities';
+import { publishTestResults } from './publishTestResults';
 
 async function run() {
     try {
         const output = getOutputPath();
         await runNLUTest(output);
         await runNLUCompare(output);
+        publishTestResults(['**/TestResult.xml'], tl.getVariable('Agent.TempDirectory'));
```

Comment:
I have a function for the search folder in this file: getCompareOutputPath().

It returns the compareOutput input if set, otherwise it will be `$(Agent.TempDirectory)/.nlu/(speech|text)/` if publishTestResults or publishNLUResults is set.